state-synchronizer/src/coordinator.rs

```diff
@@ -958,7 +925,7 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
     /// the highest ledger info in the local storage (some committed transactions are ahead of the
     /// latest ledger info and are not going to be used for helping the remote subscribers).
     /// The function assumes that the local state has been synced with storage.
-    async fn check_subscriptions(&mut self) {
+    fn check_subscriptions(&mut self) {
```

Comment:
Is the change in this function intentional?

Reply:
Yes. Because nothing inside is async at all.